src/client/datascience/notebook/notebookEditor.ts

```diff
@@ -268,16 +284,6 @@ export class NotebookEditor implements INotebookEditor {
         }
     }
 
-    private async onInterruptTimedOut(_: IKernel) {
```

Comment:
This would restart the wrong kernel. If we had two notebooks, this would result both kernels being restarted.